cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -88,6 +88,14 @@ function wait_for_istio_ready() {
     fi
 }
 
+function copy_istio_cni_conf_files() {
+  if [ "$KUBEVIRT_DEPLOY_ISTIO" == "true" ] && [ "$KUBEVIRT_WITH_CNAO" == "true" ]; then
```

Comment:
nit: please adjust indentation to the remainder of the script

Reply:
Thanks for the heads up, done